client/src/component/seviceCardHomePage/style.js

```diff
@@ -0,0 +1,21 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  media: {
+    height: 200,
+    width: 200,
```

Comment:
Can you tell me what's unit will be used when not add a specific unit?

Reply:
pexels